src/main/java/org/gradle/profiler/MavenScenarioDefinition.java

```diff
@@ -9,8 +10,17 @@
 public class MavenScenarioDefinition extends ScenarioDefinition {
     private final List<String> targets;
 
-    public MavenScenarioDefinition(String scenarioName, String title, List<String> targets, Supplier<BuildMutator> buildMutator, int warmUpCount, int buildCount, File outputDir) {
-        super(scenarioName, title, buildMutator, warmUpCount, buildCount, outputDir);
+    public MavenScenarioDefinition(
+        String scenarioName,
+        String title,
+        List<String> targets,
+        Supplier<BuildMutator> buildMutator,
+        int warmUpCount,
+        int buildCount,
+        File outputDir,
+        @Nullable  File mavenHome
```

Comment:
Adding the tool home to `ScenarioDefinition` doesn't seem like the right place and introduces some oddness with the `GradleScenarioDefinition`. I think it would be better to introduce a new type like `CommandLineBuildToolScenarioDefinition` and add the home and the command name there.

Later there would maybe be a `MavenBuildConfiguration` and a `BazelBuildConfiguration` instead of add a `File` field to `ScenarioDefinition`. The `(Gradle|Maven|Bazel)BuildConfiguration`s then can have a base class `BuildToolConfiguration` which take the `toolHome` and the `executableName` or something like that. At some point, there may also be a `JvmBuildToolConfiguration` which captures the JVM arguments and the Java home for Gradle and Maven. That is something for later, though.

Reply:
Done.